tests/functional/meson.build

```
@@ -167,7 +167,7 @@ suites = [
       'impure-env.sh',
       'debugger.sh',
       'extra-sandbox-profile.sh',
-      'help.sh',
+      #'help.sh',
```

I don't think we want to exclude it unconditionally.

```suggestion
      'help.sh',
```

---

Unfortunately one of the VM tests fails without this, see https://github.com/NixOS/nix/actions/runs/11895552419/job/33145394782.

---

Discussed in meeting
Possible solutions
- add `nix-manual.man` manpages to NixOS test `common.nix` <= preferred
- add `nix` (`everything.nix`) preferably not because that includes the functional tests
- add a meson option to disable `help.sh`

---

Changed this to test `nix --help` unconditionally at least. But I've disabled the `man` tests.